packages/worker/src/api/controllers/global/users.ts

```diff
@@ -84,12 +116,28 @@ export const adminUser = async (ctx: any) => {
 
 export const destroy = async (ctx: any) => {
   const id = ctx.params.id
+
   await users.destroy(id, ctx.user)
+
   ctx.body = {
     message: `User ${id} deleted.`,
   }
 }
 
+export const bulkDelete = async (ctx: any) => {
+  const { userIds } = ctx.request.body
+  try {
+    let { groupsToModify, usersResponse } = await users.bulkDelete(userIds)
+    await groupUtils.bulkDeleteGroupUsers(groupsToModify)
```

Comment:
I think it would be better that the group handling be done inside the users sdk, that's now an important part of the users logic. Should just be a move and can then simplify the response needed